doc/operations/configuration.rst

```diff
@@ -0,0 +1,235 @@
+Configuration
+=============
+
+The configuration for each CCF node must be contained in a single JSON configuration file specified to the ``cchost`` executable via the ``--config </path/to/configuration/file>`` argument.
+
+.. tip:: JSON configuration samples:
+
+    - Minimal configuration: https://github.com/microsoft/CCF/blob/main/samples/config/minimal_config.json
+    - Full configuration: https://github.com/microsoft/CCF/blob/main/samples/config/config.json
+
+Configuration Options
+---------------------
+
+``enclave_file``
+~~~~~~~~~~~~~~~~
+
+Path to CCF enclave application.
+
+``enclave_type``
+~~~~~~~~~~~~~~~~
+
+Type of enclave application (either ``release``, ``debug`` or ``virtual``). Default value: ``release``.
+
+``network``
+~~~~~~~~~~~
+
+The ``network`` section includes configuration for the interfaces a node listens on (both node-to-node and RPC).
+
+- ``node_address``: Address (hostname and port) to listen on for incoming node-to-node connections.
+
+- ``rpc_interfaces``: Addresses (hostname and port) to listen on for incoming client TLS connections.
+Each RPC address must contain:
+
+- The local RPC address (``rpc_address``) the node listens on.
```

Comment:
I wonder if we should go for `bind_address` and `advertised_address`?

Reply:
Done. Renamed to `bind_address` and `published_address`.